viewer/src/public/migration/Cognite3DViewer.ts

```diff
@@ -312,6 +400,10 @@ export class Cognite3DViewer {
     if (options.orthographicCamera) {
       throw new NotSupportedInMigrationWrapperError();
     }
+    if (options.onComplete) {
```

Comment:
@j-bjorne is it ok to throw on it? I didn't see any usage, so it would be implicitly ignored without throwing.